### PR TITLE
Add module variable to allow write permission

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request adds a module variable (`var.read_only`) that allows for the addition of write permissions to the IAM role being created.

## 💭 Motivation and context ##

This change is required in order to create a read-write role that allows access to a particular COOL environment's remote state.

Note also that these changes have been made in a way that is backward compatible with the role's previous behavior, so this is a non-breaking change.

## 🧪 Testing ##

All automated tests pass.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

File: locals.tf [hunk 1]
```diff
@@ -2,15 +2,33 @@ locals {
...
+  # writing.
+  object_actions_read_write = concat(
+    local.object_actions_read,
+    local.object_actions_write,
+  )
+
   # If var.role_description contains two instances of "%s", use format()
   # to replace the first "%s" with var.s3_bucket and the second "%s"
   # with var.entity_name, otherwise just use var.role_description as is
   role_description = length(regexall(".*%s.*%s.*", var.role_description)) > 0 ? format(var.role_description, var.s3_bucket, var.entity_name) : var.role_description
```
Comment (dav3r) [inline locals.tf]: I think the role description should be expanded to include a third value replacement of either "read-only" or "read-write", similar to what you did in https://github.com/cisagov/s3-read-role-tf-module/pull/47/commits/f95171913cb01e844d2afa2e1bb182a07a4bc2a6.

Even though the role name will include "ReadOnly" or "ReadWrite", I think it's preferable to document that info in the description as well.

**Review by dav3r — APPROVED** [review]
LGTM, with one request. 👍

**Review by dv4harr10 — APPROVED** [review]

File: locals.tf (hunk 1 above)
Comment (jsf9k) [reply locals.tf]: Does commit 137746d do it for you?

Comment (dav3r) [reply locals.tf]: Perfect, thank you! 🚀

**Pull request merged**